contributors/design-proposals/dynamic-admission-control-configuration.md

```diff
@@ -112,29 +116,82 @@ const (
 // AdmissionHookClientConfig contains the information to make a TLS
 // connection with the webhook
 type AdmissionHookClientConfig struct {
-	// Address of the external admission hook, could be a host string, 
-    // a host:port pair, or a URL.
-	Address string
+    // Service is a reference to the service for this webhook. It must communicate
```

Comment:
I would say-- if multiple ports are exposed, it will use 443 if available and be an error condition otherwise. If only a single port is exposed it can just use that.